src/backend/utilities/configuration.ts

```diff
@@ -78,7 +78,14 @@ const twitch = {
 if (!twitch.clientId || !twitch.secret) {
   throw new ConfigurationError('No Twitch client credentials provided');
 }
+const instagram = {
+  clientId: env.CLIENT_ID_INSTAGRAM as string,
```

Comment:
We generally prefix values in a group with a common prefix, with the exception of secrets where we put `SECRET` on the first place
```suggestion
  clientId: env.INSTAGRAM_CLIENT_ID as string,
```

Reply:
Oops, I have not been following this pattern. I thought we had decided to always put the CLIENT_ID first

Reply:
Sorry if I mislead you earlier…